arestor/client/arestor_client.py

```diff
@@ -0,0 +1,88 @@
+# Copyright 2016 Cloudbase Solutions Srl
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+import requests
+
+from arestor.client import resource as base_client
+
+
+class ArestorClient(base_client.ResourceClient):
+    """Arestor client."""
```

Comment:
I think we will need some more verbose docstrings, or taken in account as a future idea.